ncm-accounts/src/main/perl/accounts.pod

```diff
@@ -11,9 +11,9 @@ ncm-accounts: NCM component to manage the local accounts on the machine.
 
 The I<accounts> component manages the local accounts on a machine. LDAP
 authentication depends on the LDAP configuration, which is handled by
-L<ncm-authconfig>.
+L<authconfig|../authconfig/>.
```

Comment:
so how does this work with `man` as regular pod files? `authconfig` actually exists.
shouldn't we do this conversion in https://github.com/quattor/release/pull/105?